doc/dts/device_tree.rst

```diff
@@ -0,0 +1,208 @@
+.. _device-tree:
+
+Device Tree in Zephyr
+########################
+
+Introduction to Device Tree
+***************************
+
+Device tree is a way of describing hardware and configuration information
+for boards.  Device tree was adopted for use in the Linux kernel for the
+PowerPC architecture.  However, it is now in use for ARM and other
+architectures.
+
+Device tree uses a specific format to describe the device nodes in a system.
+This format is described in `EPAPR document`_.
+
+.. _EPAPR document: http://www.devicetree.org/specifications-pdf
+
+More device tree information can be found at the `device tree website`_.
+
+.. _device tree site: http://www.devicetree.org/
+
+
+System build requirements
+*************************
+
+The Zephyr device tree feature requires a device tree compiler (DTC) and Python
+YAML packages.
+
+Windows:
+  Download lbmeng/dtc and compile this in the mingw environment
+  Make sure dtc executable is in the path
+
+Linux:
```

Comment:
Same here for Linux and macOS. Instead of adding instructions here, why not redirect to a section in the Getting Started guide where Device Tree installation is described?